interventions/models.py

```diff
@@ -6,26 +6,33 @@ class Intervention(models.Model):
     name = models.CharField('Nazwa', max_length=255, default="zabieg", null=False)
     description = models.TextField('Opis')
     min_time = models.FloatField('Czas trwania zabiegu')
+    def __str__(self):
+        return self.name
 
 
 class Operation(models.Model):
     intervention = models.ForeignKey(Intervention, on_delete=models.CASCADE)
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     date = models.DateField()
     time = models.TimeField()
+    def __str__(self):
+        return self.intervention
+
 
 class Doctor(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     specialization = models.CharField(max_length=64)
     section = models.CharField(max_length=64)
     card_number = models.IntegerField('Id lekarza')
+    def __str__(self):
+        return self.user
```

Comment:
Pytanie jak wyświetla się user. Sprawdziłeś? Proponuję dążyć do formy "Dr {imie} {nazwisko}"

Reply:
Jak mogę to sprawdzić nie w chodząc w panel admina? w admin django?

Reply:
Możesz wejść a panel admina, to będzie dobra i szybka droga, musisz tylko zrobić superusera (szukaj: python manage.py createsupseruser). Jeżeli nie to możesz wyprintować listę userów (user.objects.all()) w jakiejś templace (czego nie polecam) albo wyprintować go w shellu (python manage.py shell).

Reply:
Zrobiłem tak że w modelu user(tym w django) dodałem 
    def __str__(self):
        return self.get_full_name()
nwm czy to jest poprawne, nie potrafiłem tego zrobić w modelach naszych xD więc proszę o odzew

Reply:
To raczej nie przejdzie, bo to co edytujesz w Django zostaje na Twoim localu bo cały env znajduje się u Ciebie i jest w gitignore (a nawet jakby nie był to to dalej zła praktyka zmieniać coś bezpośrednio we frameworku). Rozkminimy ten problem na spotkaniu bo sam nie wiem jak to zrobić

Reply:
Albo spróbuj jeszcze tego: https://stackoverflow.com/questions/34214320/django-customize-the-user-models-return-field

Reply:
Próbowałem nie działało